src/argrelay/plugin_interp/InterpTreeInterp.py

```diff
@@ -147,7 +147,7 @@ def propose_arg_completion(self) -> None:
                 if (
                     isinstance(x, str)
                     and
-                    x != default_tree_leaf_
+                    x != surrogate_node_id_
```

Comment:
Should it be `surrogate_tree_leaf_`?

Reply:
No, `x` is a `node_id` (`dict` key, not `dict` value).